### ISSUE TITLE
"git repository is invalid" when attempting to run DALL-E Mega on Windows

### ISSUE THREAD

**Issue by DigitalDwagon** [saharmor/dalle-playground#19]
When trying to run DALL-E mega on windows, I get the following error in logs:
```
2022-06-07 18:40:23,810 DEBUG   HandlerThread:20932 [meta.py:probe():209] probe
2022-06-07 18:40:23,821 DEBUG   HandlerThread:20932 [git.py:repo():29] git repository is invalid
```
After we run it, everything looks good until it finishes downloading the mega model, then it just spits me back to the cursor with no further information. I found that error after looking through the logs. Me and a friend have tried several troubleshooting steps (including throwing everything out and starting over) but we'e not sure where to go from here. We were able to get DALL-E Mini running correctly, so we think we should be able to get this too with some help.

Thanks all!

**Comment by saharmor** [comment]
@DigitalDwagon can you please post the exact steps you took so I can reproduce this on my end?

**Comment by DigitalDwagon** [comment]
Fixed with #26 and #29
